dolphinscheduler-worker/src/main/java/org/apache/dolphinscheduler/server/worker/runner/RetryReportTaskStatusThread.java

```diff
@@ -58,25 +57,25 @@ public void start(){
      */
     @Override
     public void run() {
-        ResponceCache responceCache = ResponceCache.get();
+        ResponseCache instance = ResponseCache.get();
```

Comment:
Keep the variable name `responseCache` might be better.

Reply:
I change it .please look at the new commit.